### PR TITLE
Mongodb overview jsonnet

### PR DESCRIPTION
**Pull request by mrinal1209**
Add support for MongoDB Overview Dashboard

@adivinho  @puneet0191

### CODE REVIEW

**Review by adivinho — CHANGES_REQUESTED** [review]
The parameter "Null value" has to be "null as zero" in the dashboards "Query Efficiency" and "Scanned and Moved Objects".
Legend -> Decimals parameter has to be 2 in the dashboard "Query Efficiency".

**Comment by mrinal1209** [comment]
> The parameter "Null value" has to be "null as zero" in the dashboards "Query Efficiency" and "Scanned and Moved Objects".
> Legend -> Decimals parameter has to be 2 in the dashboard "Query Efficiency".

@adivinho  changes made please review

**Comment by adivinho** [comment]
Please disable time picker hiding.
Screenshot_20190712_181458

**Comment by mrinal1209** [comment]
@adivinho changes made please review

File: pmm-templates/MongoDB_Overview.jsonnet [hunk 1]
```diff
@@ -22,6 +22,9 @@ dashboard.new(
   tags=[ "MongoDB","Percona"],
   iteration=1553588305920,
   uid="6Lk9wMHik",
+  timepicker = timepicker.new(
+    hidden = true,
```
Comment (adivinho) [inline pmm-templates/MongoDB_Overview.jsonnet]: Now it's hidden but it has to be shown.

**Review by adivinho — CHANGES_REQUESTED** [review]

File: pmm-templates/MongoDB_Overview.jsonnet (hunk 1 above)
Comment (mrinal1209) [reply pmm-templates/MongoDB_Overview.jsonnet]: I don't understand what does that mean??

Comment (adivinho) [reply pmm-templates/MongoDB_Overview.jsonnet]: The TimePicker element is hidden in your template.
Screenshot_20190717_124057
It has to be shown.
e.g.
Screenshot_20190717_123958

**Comment by mrinal1209** [comment]
@adivinho @puneet0191  I have disabled the editable field and reverted the time picker kindly review again.

**Comment by adivinho** [comment]
Please fix extra spaces in descriptions. 

Screenshot_20190723_223815

**Comment by mrinal1209** [comment]
@adivinho please help to test and review the changes

**Review by adivinho — APPROVED** [review]

**Pull request merged**